teuthology/test/test_proc_regex.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+
+from teuthology.orchestra import cluster
+from teuthology import misc
+from teuthology.orchestra.daemon.group import DaemonGroup
+import subprocess
+
+
+class FakeRemote(object):
+    pass
+
+
+def test_get_daemon_pid():
+    ps_output = """
+    ceph       658     1  0 Jun08 ?        00:07:43 /usr/bin/ceph-mgr -f --cluster ceph --id host1 --setuser ceph --setgroup ceph
+    ceph      1634     1  0 Jun08 ?        00:02:17 /usr/bin/ceph-mds -f --cluster ceph --id host1 --setuser ceph --setgroup ceph
+    ceph     31555     1  0 Jun08 ?        01:13:50 /usr/bin/ceph-mon -f --cluster ceph --id host1 --setuser ceph --setgroup ceph
+    ceph     31765     1  0 Jun08 ?        00:48:42 /usr/bin/radosgw -f --cluster ceph --name client.rgw.host1.rgw0 --setuser ceph --setgroup ceph
+    ceph     97427     1  0 Jun17 ?        00:41:39 /usr/bin/ceph-osd -f --cluster ceph --id 0 --setuser ceph --setgroup ceph
```

Comment:
The contents of the excerpt is better to put to a separate file, and read it using relative path `os.path.dirname(__file__)`

Reply:
@kshtsk, can you please suggest me the path to place this file.

Reply:
updated.